runtime/command-start/src/main/java/io/aklivity/zilla/runtime/command/start/internal/airline/ZillaStartCommand.java

```diff
@@ -133,6 +133,7 @@ public void run()
         try (Engine engine = Engine.builder()
             .config(config)
             .errorHandler(this::onError)
+            .report(report)
```

Comment:
I was thinking wether it's the best to propagate the consumer here. Maybe we just log everytime? Or since the reconfigure method will be called at initial configuration as well the try-catch here can be even removed. I think that'd be the best option.

Reply:
Can we use the existing error handler instead of introducing `report`?